js/dataManager.js

```diff
@@ -26,13 +26,34 @@ export class DataManager{
           acc[counter].question = question;
           acc[counter].answer = answer;
           acc[counter].value = pointValue;
-          acc[counter].category = currentCategory;//categories are camelcase
+          acc[counter].category = this.parseTitle(currentCategory);//categories are camelcase
          
           counter++
         }
       
        return acc;
        }, { })
       }
+
+    parseTitle(str) {
+
+      var strToArray = str.split('');
+      var capitalIndexes = [];
+      
+      strToArray.forEach((char) => {
+          if (char === char.toUpperCase()) {
+              capitalIndexes.push(strToArray.indexOf(char));
+          }
+      });
+
+      capitalIndexes.forEach((capital, i) => {
+          let notCapitalWord = (capitalIndexes.indexOf(capital-1) === -1);
+          if (notCapitalWord) strToArray.splice(capital + i, 0, ' ');
+          i++;
+      });
+
+      strToArray[0] = strToArray[0].toUpperCase();
       
+      return strToArray.join('');
+  }
```

Comment:
Nice, I like that all of this is a helper method so that you can cleanly call it up top with `this.parseTitle`